independent-projects/bootstrap/pom.xml

```diff
@@ -324,8 +325,13 @@
             </dependency>
             <dependency>
                 <groupId>org.jboss.slf4j</groupId>
-                <artifactId>slf4j-jboss-logging</artifactId>
-                <version>${slf4j-jboss-logging.version}</version>
+                <artifactId>slf4j-jboss-logmanager</artifactId>
+                <version>${slf4j-jboss-logmanager.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j-api.version}</version>
             </dependency>
```

Comment:
Why introduce this dependency here?

Reply:
In `slf4j-jboss-logmanager` it's scope is provided.
Tried without and the test is failing.

Reply:
Ah, I see; it was an implicit dependency before.  Thanks.